wagtail/admin/viewsets/model.py

```diff
@@ -87,6 +90,9 @@ class ModelViewSet(ViewSet):
     #: The fields to exclude from the inspect view.
     inspect_view_fields_exclude = []
 
+    #: Whether to enable the copy view. Defaults to ``True``.
+    copy_view_enabled = True
```

Comment:
Just a drive by comment - I wonder if we need this `test_model_viewset`. Why not just set the `copy_view_class` to `None` if you do not want to use it?

However, maybe we want the default to be no copy view enabled, this seems like it may not be something wanted by all models.

I will let others give their thoughts on this though.

Reply:
I assume that you mean `copy_view_enabled`  instead of  `test_model_viewset`.

A couple thoughts:

1. Having a Boolean to set this is easier for me to set from `True` to `False` since I can easily look in the code where it says `False` and change it to `True`. If the `copy_view_class` is set to `None` I will have to find in the documentation what it should be set to. This is probably more true if the default is copy_view_enabled=False.
2. ModelViewSet enables and disables the inspect view by the Boolean `inspect_view_enabled` and it also has a `inspect_view_class` attribute. It might be confusing for the copy view to work differently. On the other hand this is not true for all of the add, edit, delete, usage, and history views. I don't know if setting these view classes to `None` will disable them or not. If setting these view_classes to `None` does disable them, then the documentation should probably reflect that and the inspect and copy view should both be disabled by setting the view class to `None`.
3. From the discussion in #10921 It seemed like the copy view would be used more often and it is less likely for someone to want it specifically off.

Reply:
Hey @lb-, it was Matt and I in conversation at https://github.com/wagtail/wagtail/issues/10921#issuecomment-1822687087 where the flag suggestion came. It was then my suggestion to model it on the existing view logic.

Reply:
Thanks for the info here.